crates/uv/src/bin/uvx.rs

```diff
@@ -0,0 +1,39 @@
+use std::{
+    ffi::OsString,
+    process::{Command, ExitCode, ExitStatus},
+};
+
+use anyhow::bail;
+
+fn run() -> Result<ExitStatus, anyhow::Error> {
+    let current_exe = std::env::current_exe()?;
+    let Some(bin) = current_exe.parent() else {
+        bail!("Could not determine the location of the `uvx` binary")
+    };
+    let uv = bin.join("uv");
+    let args = ["tool", "run"]
```

Comment:
In the future, this might invoke a hidden `uv x` so we can special case the help menu and such?